runtime/lua/vim/_comment.lua

```diff
@@ -0,0 +1,264 @@
+---@nodoc
+---@class vim._comment.Parts
+---@field left string Left part of comment
+---@field right string Right part of comment
+
+--- Get 'commentstring' at cursor
+---@param ref_position integer[]
+---@return string
+local function get_commentstring(ref_position)
```

Comment:
Possible followup: this could be more broadly useful refactored as a `vim.treesitter.get_local_option('commentstring', position)`.